runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierConfig.java

```diff
@@ -70,6 +72,7 @@ default boolean logoutIdle()
 		description = "The notification delay after the player is idle",
 		position = 4
 	)
+	@Range(min = ANIMATION_IDLE_MINIMUM)
```

Comment:
This will also affect the interaction idle notification. Isn't it enough to just do what you've done in `IdleNotififerPlugin.java` so that it does not respect any configs below 601 for animations?

I need my instant fishing interaction notifications. :P